packages/components/package.json

```diff
@@ -107,7 +107,7 @@
     "rollup-plugin-ts": "^3.4.5",
     "stylelint": "^16.3.1",
     "stylelint-config-rational-order": "^0.1.2",
-    "stylelint-config-standard-scss": "^13.1.0",
+    "stylelint-config-standard-scss": "^14.0.0",
```

Comment:
Noting here that this major is a shift in `stylelint` support window, requiring styling 16.11+.
We formally upgrade `stylelint` in [2623](https://github.com/hashicorp/design-system/pull/2623/files#diff-2c76dc06185f93bec73660e15338433b95eac6707317564c0f778f5b9abe446cR108), although in `main` it resolves in `16.11.0` anyway at the moment (because of other dependencies requiring this version), so all good.